index.html

```diff
@@ -196,40 +207,54 @@ <h2>About</h2>
-          <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
-          <button class="" type="submit" name="submit-btn">Start collaboration</button>
-      </form>
-
-      <div class="bottom_row">
-        <a class="text-button text_body_font resume_color">
-            <p>Get my Resume</p>
-            <div class="arrow_upload"></div>
-        </a>
-
-        <div class="divicon">
-            <ul class="sociallist">
-           <li class="sociallist_link"><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/linkdin.png" alt="linkdin"></a></li>
-           <li class="sociallist_link" ><a href="#"><img src="img/medium.png" alt="medium"></a></li>
-           <li class="sociallist_link"><a href="#"><img src="img/github.png" alt="github"></a></li>
-           
-            </ul>
+    
+        <div class="contact_div">
+
+    
+            <h4 class="title-contact text_header_font">Get Started</h4>
+
+            <p class="section-desc text_body_font">
+                Hello!I am a software developer! I can 
+                help you build a product, feature or website.
+                Take a look of my works. If you like what you see 
+                and have a project you need coded, don't hesitate 
+                and contact me.
+            </p>
+            <form action="https://formspree.io/f/mzbovllg" method="post">
+                <input type="text" name="FullName" placeholder="Enter Your Name" maxlength="30" required>
+                <input type="email" name="Email" placeholder="Enter Your Email" required>
+                <textarea name="Message" placeholder="Enter Your Message"  maxlength="500" required></textarea>
+                <button class="" type="submit" name="submit-btn">Start collaboration</button>
+            </form>
         </div>
 
-      </div>
+                <div class="bottom_row">
+                    <div class="get_cv">
+                        <a class="text-button text_body_font resume_color">
+                            <p class="arrow_upload">Get my Resume</p>
+                            <div ></div>
+                        </a>
+                    </div>
+                    
+
+                    <div class="divicon">
+                        <ul class="sociallist">
+                    <li class="sociallist_link"><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
+                    <li class="sociallist_link" ><a href="#"><img src="img/linkdin.png" alt="linkdin"></a></li>
+                    <li class="sociallist_link" ><a href="#"><img src="img/medium.png" alt="medium"></a></li>
+                    <li class="sociallist_link"><a href="#"><img src="img/github.png" alt="github"></a></li>
+                    
+                        </ul>
+                    </div>
+
+                </div>

```

Comment:
- [ ] Kindly make your Footer match the Figma design.
- [ ] Kindly add some padding to the left and the right.
- [ ] Kindly make it fit the page width.


Your design
![image](https://user-images.githubusercontent.com/35535130/160385978-e62cd130-960e-40ef-82d2-eb13813963ec.png)


Figma design
![image](https://user-images.githubusercontent.com/35535130/160386011-dc93cb93-c824-47f3-9f50-852917d0f2df.png)